runtime/polkadot/src/lib.rs

```diff
@@ -136,55 +136,55 @@ pub fn native_version() -> NativeVersion {
 }
 
 pub struct BaseFilter;
-impl Contains<Call> for BaseFilter {
-	fn contains(call: &Call) -> bool {
+impl Contains<RuntimeCall> for BaseFilter {
```

Comment:
I think this is actually not needed anymore.

Reply:
https://github.com/paritytech/polkadot/pull/5969

Reply:
so should I just revert this?